### PR TITLE
Jordan/action modal cancel button

### PR DESCRIPTION
**Pull request by jbibla**
**Description:**

- swapped the little x to close the action modal for a proper button

Thank you! 🚀

---

For contributor:

- [ ] Added changes entries. Run `yarn changelog` for a guided process.
- [ ] Reviewed `Files changed` in the github PR explorer
- [ ] Attach screenshots of the UI components on the PR description (if applicable)
- [ ] Scope of work approved for big PRs

For reviewer:

- [ ] Manually tested the changes on the UI

### CODE REVIEW

**Comment by faboweb** [comment]
Should this include a screenshot or are we happy with the preview?

File: changes/jordan_action-modal-cancel-button [hunk 1]
```diff
@@ -0,0 +1,2 @@
+[Changed] switched the little x on the action modal to be a proper button @jbibla
+[Changed] made the edit memo button a link for a cleaner send modal @jbibla
```
Comment (faboweb) [inline changes/jordan_action-modal-cancel-button]: these should be separate PRs

**Comment by jbibla** [comment]
> Should this include a screenshot or are we happy with the preview?

was the end of my day and i didn't want to wait for netlify. was planning on sharing a screenshot this morning 😄

**Comment by jbibla** [comment]
my other PR adds more padding to the action-modal-footer:
Screen Shot 2019-12-11 at 8 18 45 AM
too many buttons was the reason for this change - but i also think it's clearer and more friendly:
Screen Shot 2019-12-11 at 8 21 29 AM
my other PR resolves this input-suffix issue:
Screen Shot 2019-12-11 at 8 20 26 AM

**Comment by faboweb** [comment]
Hmm, I prefer the old denom in input stlye. Why did you pull it out?

**Comment by jbibla** [comment]
> Hmm, I prefer the old denom in input stlye. Why did you pull it out?

LOL. no, this is a bug - i fixed it in the other PR! 

https://github.com/luniehq/lunie/pull/3284/files#r356585170
